chunkset_tpl.h

```diff
@@ -171,5 +171,8 @@ Z_INTERNAL uint8_t* CHUNKMEMSET_SAFE(uint8_t *out, unsigned dist, unsigned len,
         }
         return out;
     }
-    return CHUNKMEMSET(out, dist, len);
+    if (len)
+        return CHUNKMEMSET(out, dist, len);
+
+    return out;
```

Comment:
Hah, interesting that the coverage check shows this as never being hit until we revert the window layout.  It definitely explains why we don't hit this issue until then.

Reply:
That's why some bugs can take years to manifest and are hard to reproduce... Sometimes it's matter of hitting cascade event before all bugs are found.